src/coll.ts

```diff
@@ -149,6 +154,8 @@ class Coll extends LitElement {
 
   private observer?: IntersectionObserver;
 
+  private archiveInfoDialog: Ref<SlDialog> = createRef();
```

Comment:
Didn't know about this lit function! Here its mostly used to just show/hide the dialog, right?
I guess another approach that we've used is having a separate bool that toggles if the dialog is rendered or not, but this is useful for referring to the element directly for other reasons!

Reply:
Yep! In my mind this is more straightforward than keeping track of state ourselves here, which would also involve listening for a dialog close event. We don't actually need to know if the dialog is open or not here!

Reply:
Yeah, I guess a pattern we've used before is something like:
```
this.isDialogOpen ? html`<sl-dialog @sl-hide=${() => this.isDialogOpen = false}> ... <sl-button @click="${() => this.isDialogOpen = false}"> ... </sl-dialog>` : ''`
```
I guess either way a member variable is maintained, though only a bool in this case, and yes, need to catch all the places dialog can be closed. Don't have a strong preference either way.